script/playbackstrategy/msestrategy.js

```diff
@@ -390,8 +394,12 @@ define('bigscreenplayer/playbackstrategy/msestrategy',
           return isEnded;
         },
         isPaused: isPaused,
-        pause: function () {
+        pause: function (opts) {
           mediaPlayer.pause();
+          opts = opts || {};
+          if (opts.disableAutoResume !== true && windowType === WindowTypes.SLIDING) {
```

Comment:
You have added the opts.disableAutoResume logic. Is this even used? if so, probably best you write a unit test for this bit of logic.

Reply:
I left it in to keep consistency between the multiple playback strategy pause methods, but can be removed if necessary.